appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -1698,6 +1751,18 @@ Blockly.ReplMgr.rendezvousDone = function() {
     });
 };
 
+Blockly.ReplMgr.makeIncompatibleDialogMsg = function(iosIncompatible, isIos){
+        var result = "<ul>"
+        for(var i=0;i<iosIncompatible.length;i++){
+            result+="<li>"+iosIncompatible[i]+"</li>"
+        }
+        result+="</ul>"
+        if(isIos){
+        result+= "<a href="+Blockly.Msg.REPL_IOS_COMPANION_PROGRESS_URL+">"+Blockly.Msg.REPL_IOS_COMPANION_PROGRESS_TRACK+"</a>"
+        }
+        return result
```

Comment:
I'm not sure why the initial indent for this function is so large. For this file it's 4 spaces.